src/libraries/Common/tests/System/IO/WrappedStream.cs

```diff
@@ -1,9 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.IO;
-
-namespace System.Formats.Tar
+namespace System.IO
 {
     public class WrappedStream : Stream
```

Comment:
@carlossanlop do you remember which changes you meant back in https://github.com/dotnet/runtime/pull/67883#discussion_r847924992

It seems to work with the consolidated version.